hazelcast-integration/eureka/springboot-embedded/README.md

```diff
@@ -106,6 +106,45 @@ Hazelcast members should form a cluster together as in the previous section. You
 
 ![Eureka Hazelcast Separate Client](markdown/eureka-hazelcast-separate-client.png)
 
+## 2.3 Hazelcast reusing Eureka Client with Metadata
+
+Sometimes, you may not want to have Hazelcast registered as a separate application in Eureka. After all, Hazelcast is not a separate application, but a library embedded inside your Spring Boot application. In that case the Eureka plugin provides a solution to store the information about Hazelcast in Metadata.
```

Comment:
Just for reader's clarity, what about elaborating a bit more on this mechanism?
E.g. something like: "...to store the information about Hazelcast _host and port in the Metadata of the application itself, by using the same Eureka client as the application._"

Reply:
Good idea. Updated according to your suggestion.